ofrak_patch_maker/ofrak_patch_maker_test/test_toolchain_c.py

```diff
@@ -121,10 +130,16 @@ def test_hello_world(toolchain: ToolchainVersion, proc: ProgramAttributes, exten
     source_path = os.path.join(source_dir, "hello_world.c")
     build_dir = tempfile.mkdtemp()
 
+    if toolchain in [ToolchainVersion.GNU_AVR_5]:
+        relocatable = False
+        base_symbols = {"__mulhi3": 0x1234}  # Dummy address to fix missing symbol
```

Comment:
Where does this come from? `*` usage in the test source?

Reply:
Yes, this is due to the use of the multiply operator in the `hello_world.c` source file. AVR requires compiler builtins for certain operations, such as 16 bit multiplication: https://gcc.gnu.org/wiki/avr-gcc#Exceptions_to_the_Calling_Convention

Reply:
A long term goal is to be able to run/emulate these test programs to completion. This issue/mitigation would result in a crash.

But, seeing as currently we are only validation the program build and patch creation, let's table that. I'll create an issue shortly to quantify this debt.

Thank you for noting this here, @rbs-afflitto.